<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quick_reservation</class>
 <widget class="QMainWindow" name="quick_reservation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgba(0, 0, 0, 150); /* 黑色边框，150为不透明度 */
    background-color: rgba(255, 255, 255, 200); /* 白色背景，200为不透明度 */
    border-radius: 10px; /* 圆角边框 */</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>340</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QPushButton" name="queding">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>340</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid rgba(0, 0, 0, 150); /* 黑色边框，150为不透明度 */
    background-color: rgba(255, 255, 255, 200); /* 白色背景，200为不透明度 */
    border-radius: 10px; /* 圆角边框 */</string>
     </property>
     <property name="text">
      <string>手机号：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="phonenumber">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid rgba(0, 0, 0, 150); /* 黑色边框，150为不透明度 */
    background-color: rgba(255, 255, 255, 200); /* 白色背景，200为不透明度 */
    border-radius: 10px; /* 圆角边框 */</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>301</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>仿宋</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid rgba(0, 0, 0, 150); /* 黑色边框，150为不透明度 */
    background-color: rgba(255, 255, 255, 200); /* 白色背景，200为不透明度 */
    border-radius: 10px; /* 圆角边框 */</string>
     </property>
     <property name="text">
      <string>快捷预约</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid rgba(0, 0, 0, 150); /* 黑色边框，150为不透明度 */</string>
     </property>
     <property name="text">
      <string>入住时间：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="intime">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>220</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>10</y>
       <width>191</width>
       <height>161</height>
      </rect>
     </property>
     <property name="text">
      <string>提示：入住时间格式00000000一二位表示月份，三四位表示日期，五六位表示时，七八位表示分</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>170</y>
      <width>191</width>
      <height>271</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgba(0, 0, 0, 150); /* 黑色边框，150为不透明度 */
    background-color: rgba(255, 255, 255, 200); /* 白色背景，200为不透明度 */
    border-radius: 10px; /* 圆角边框 */</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> color: black; /* 设置字体颜色为白色 */
</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> color: black; /* 设置字体颜色为白色 */
</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> color: black; /* 设置字体颜色为白色 */
</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> color: black; /* 设置字体颜色为白色 */
</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> color: black; /* 设置字体颜色为白色 */
</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> color: black; /* 设置字体颜色为白色 */
</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> color: black; /* 设置字体颜色为白色 */
</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> color: black; /* 设置字体颜色为白色 */
</string>
     </property>
     <property name="text">
      <string>9</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> color: black; /* 设置字体颜色为白色 */
</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> color: black; /* 设置字体颜色为白色 */
</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>回退</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>210</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>完成</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
